assets/styles/style.css

```diff
@@ -0,0 +1,722 @@
+    flex-wrap: wrap;
+  }
+
+  .card-1,
+  .card-2,
+  .card-3,
+  .card-4,
+  .card-5 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: rgba(252, 241, 241, 0.277);
+    padding: 30px;
+    width: 250px;
+    margin-left: 0;
+    margin-right: 2px;
+    margin-bottom: 100px;
+    height: 72%;
+  }
+
+  .card-4 {
+    border: 4px solid var(--secondary_1);
+  }
+
+  .card-flex {
+    display: flex;
+    flex-direction: row;
+    gap: 0;
+    justify-content: center;
+    background-image: url(../images/xpattern.jpg);
+    width: 100%;
+    background-repeat: repeat;
+    background-size: 10%;
+    background-color: var(--primary_3);
+    height: 100%;
+  }
+
+  .join-event-btn {
+    display: none;
+  }
+
+  .mobile-second-page p {
+    color: var(--secondary_1);
+    font-size: 20px;
+    text-align: center;
+    width: 100%;
+    height: 97%;
+  }
+
+  .mobile-second-page h3 {
+    color: var(--primary_2);
+    font-size: 20px;
+    text-align: center;
+    width: 39%;
+  }
+
+  .mobile-second-page a {
+    color: var(--secondary_1);
+    font-size: xx-large;
+    text-align: center;
+    width: 25%;
+  }
+
+  .mobile-second-page h2 {
+    font-family: 'Lato', sans-serif;
+    font-size: 154%;
+    font-weight: 800;
+    color: var(--secondary_1);
+    margin: 0% 0% 0% 44%;
+  }
+
+  .mobile-second-page h2::after {
+    content: "";
+    background-color: var(--primary_2);
+    display: block;
+    width: 11%;
+    border-radius: 0%;
+    margin: 2% 0% 0% 5%;
+  }
+
+  .mobile-second-page .all-program-link {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 60px;
+    font-size: 10px;
+    color: var(--secondary_1);
+  }
+
+  .speaker-1 {
+    width: 35%;
+    margin: 0;
+  }
+
+  .image-display-1 {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 12%;
+    margin-left: 22%;
+    margin-top: 4%;
+  }
+
+  .mobile-third-page .more-btn {
+    display: none;
+  }
+
+  .partners {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 0%;
+    background-color: #424146;
+    margin-bottom: 0%;
+    margin-top: 17%;
+  }
+
+  .partners h2 {
+    color: #919191;
+  }
+
+  .partners h2::after {
+    content: "";
+    background-color: #919191;
+    display: block;
+    width: 1.5em;
+    height: 2px;
+    border-radius: 0.5px;
+    margin: 15px auto;
+  }
+
+  .partners-logo {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    filter: opacity(0.5) drop-shadow(0 0 0 #fff);
+  }
+
+  .partners-logo .github {
+    margin-left: 50px;
+    margin-right: 50px;
+  }
+
+  .partners-logo .microverse {
+    margin-left: 40px;
+  }
+
+  .mobile-about-footer {
+    background-color: var(--secondary_2);
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: 0 0 0 0;
+    width: 100%;
+    height: 76px;
+  }
+
```

Comment:
- [ ] As requested by the previous reviewer, the footer on the about page should have a darker background, kindly consider implementing this so your page looks exactly as the provided design. Please make sure to use the same color as the one on provided Behance design.

   - ### YOUR DESIGN
      ![image](https://user-images.githubusercontent.com/87197412/205362501-44596d97-2f94-4c1b-85f5-ce751f62c251.png)

   - ### PROVIDED DESIGN
   
      ![image](https://user-images.githubusercontent.com/87197412/205362433-83d6e5a5-21d3-4a16-beb4-e86a494d2575.png)